includes/components/class-admin.php

```diff
@@ -908,7 +908,7 @@ function( $extension ) use ( $referral ) {
 			$extension_status = install_plugin_install_status( $extension );
 
 			// Set activation status.
-			if ( ! in_array( $extension_status['status'], [ 'install', 'update_available' ], true ) && ! is_plugin_active( $extension_path ) ) {
+			if ( $extension_status['file'] && ! is_plugin_active( $extension_path ) ) {
```

Comment:
Please explain how this fixes the issue, is this was specific to WordPress.org? What was the difference between the extensions installed via the TGMPA notice and directly via ZIP?

Reply:
The issue that `install_plugin_install_status` returns plugin filename relative to the plugins directory as element of array only when plugin is installed but not activated. But in this case plugin still has status `install` which is not correct for current condition so the condition was changed